Examples/Swift/ViewController.swift

```diff
@@ -298,21 +300,32 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
     }
     
     func setupMapView() {
-        guard mapView == nil else { return }
-        mapView = NavigationMapView(frame: view.bounds)
+        guard self.mapView == nil else { return }
+        let mapView = NavigationMapView(frame: view.bounds)
+        self.mapView = mapView
         
-        mapView!.autoresizingMask = [.flexibleWidth, .flexibleHeight]
-        mapView!.delegate = self
-        mapView!.navigationMapDelegate = self
-        mapView!.userTrackingMode = .follow
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        mapView.navigationMapDelegate = self
+        mapView.userTrackingMode = .follow
+        let bottomPadding = (view.frame.height + view.frame.origin.y) - bottomBar.frame.origin.y
         
-        view.insertSubview(mapView!, belowSubview: bottomBar)
+        var topPadding: CGFloat = 0.0
+        if #available(iOS 11.0, *) {
```

Comment:
There is a convenient `safeTopAnchor` in an extension on UIView. Can we make use of that instead?

Reply:
This is setting up the map inset, not an auto layout constraint.

Reply:
- [ ] Try and see if this can be done better with Storyboarding on `ViewController`

Reply:
Is this still actionable?

Reply:
@bsudekum @frederoni Looked into this -- bringing the creation of the mapView into storyboard doesn't look like a good idea right now, as the storyboard will likely keep a reference to it while the navigation scene is presented. There may be ways to get around this, but it's non-trivial enough to look out-of-scope for this PR.